source/.gitignore

```diff
@@ -14,3 +14,7 @@
 # Ignore all logfiles and tempfiles.
 /log/*.log
 /tmp
+
+# Ignore vim tempfiles
```

Comment:
if you haven't already, you may want to create a global .gitignore for stuff like vim tempfiles & .DS_Store...

See: https://help.github.com/articles/ignoring-files/